tsconfig.json

```diff
@@ -66,5 +66,11 @@
     /* Advanced Options */
     "skipLibCheck": true,                     /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true  /* Disallow inconsistently-cased references to the same file. */
-  }
+  },
+  "include": [
+    "src/**/*"
+  ],
+  "exclude": [
+    "test",
```

Comment:
This trailing comma should make it invalid JSON. Check your IDE validation settings.